### PR TITLE
Travis Perf - Stop spinning up new env just for e2e

### PR DESCRIPTION
**Pull request by tgandrews**
### What is the context of this PR?
Stop spinning up new env especially for e2e tests

### How to review 
1. Ensure that we do not spin up a new env in travis for the e2e

### CODE REVIEW

**Comment by tgandrews** [comment]
[Review Travis build times](https://trello.com/c/Jj70qEVd/825-review-travis-build-times)

File: eq-author/cypress/integration/e2e/e2e_spec.js [hunk 1]
```diff
@@ -61,18 +62,22 @@ describe("End to end", () => {
...
+    const fieldsToIgnore = ["id", "form_type", "eq_id", "survey_id"];
+    cy.request(
+      `${Cypress.env("PUBLISHER_URL") ||
+        "http://localhost:9000"}/publish/${questionnaireId}`
+    ).then(response => {
+      // Use this to grab a copy of what was sent from publisher
+      // cy.writeFile(
+      //   "cypress/fixtures/publisher_new.json",
+      //   JSON.stringify(response.body, null, 2)
+      // );
```
Comment (jonnyshaw89) [inline eq-author/cypress/integration/e2e/e2e_spec.js]: Should we just delete this code?

Comment (tgandrews) [reply eq-author/cypress/integration/e2e/e2e_spec.js]: Its left so when people need to update the expected output the code is there to do it. Basically to make it as easy as possible to keep adding things to the build questionnaire and then not having to worry about this bit. I found it non-obvious that you had to use the cypress methods and not jut normal node methods.

File: eq-author/scripts/run_tests.sh [hunk 2]
```diff
@@ -39,15 +39,13 @@ if [ "$1" == "integration" ]; then
...
-else
-  integration_folder="cypress/e2e";
 fi
 
 cypress_config="watchForFileChanges=false,integrationFolder=$integration_folder"
 
 # Run the tests
 if [ -z "${CYPRESS_RECORD_KEY-}" ]; then
-  yarn cypress run --browser chrome --config "$cypress_config"
+  CYPRESS_PUBLISHER_URL=http://localhost:19000 CYPRESS_API_URL=http://localhost:14000 yarn cypress run --browser chrome --config "$cypress_config"
```
Comment (samiwel) [inline eq-author/scripts/run_tests.sh]: Shouldn't the CYPRESS_API_URL include the `/graphql`?

Comment (samiwel) [reply eq-author/scripts/run_tests.sh]: Ignore comment, I misread the code.

**Pull request merged**